scripts/west_commands/sign.py

```diff
@@ -45,9 +45,8 @@
 imgtool
```

Comment:
Nothing wrong with this line, but GitHub won't let me comment on line 36 since it's not part of the diff.

However, line 36 mentions zephyr.signed.bin and zephyr.signed.hex, which rimage does not appear to use. Is that right? If so, does it make sense to fix that line too?

Reply:
Yes, rimage tool does not use zephyr.signed.bin and zephyr.signed.hex. Shall I remove that text?

Reply:
How about moving this to the `imgtool` specific section, and then at line 36, refer with some as:
> See detailed signing tool description for signed binaries created.

Reply:
Nevermind.
See you did this in the following commit.
Thanks.